components/log/test/index.test.mjs

```diff
@@ -0,0 +1,12 @@
+import "../../__fixture__.mjs";
+
+import {
+  logWarning,
+  logWarningWhen,
+  logInfo,
+} from "./index.mjs?env=test&log-level=warning";
+
+logInfo("foo");
+logWarning("bar");
+logWarningWhen(true, "qux");
+logWarningWhen(false, "qux");
```

Comment:
Yeah, it is not straightforward to check the message is written to the log file because the test version of `log-inner` is a noop. I thought the tested code was simple enough that I could skip it.